tmp/test.rs

```diff
@@ -0,0 +1,15 @@
+fn main() {
```

Comment:
Could you remove this file, I'm guessing this was included in the commit by mistake :smile: